src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -54,12 +54,18 @@ interface AstroIntegration {
         logger: AstroIntegrationLogger;
     }) => void | Promise<void>;
     'astro:build:done'?: (options: { dir: URL; routes: RouteData[]; logger: AstroIntegrationLogger; }) => void | Promise<void>;
+
+    // ... integration-defined hooks gets added here
```

Comment:
```suggestion
    // ... any custom hooks from integrations
```
I like this matching the language that's already on the page. Since most people won't have to worry about this, I like this looking like, "oh, and maybe some stuff will get added here" with a direct heading ("Custom hooks") they can go see what it means, if they're curious.